taichi/ir/ir.cpp

```diff
@@ -7,6 +7,7 @@
 #include <unordered_map>
 
 #include "taichi/ir/frontend.h"
+#include "taichi/ir/frontend_ir.h"
```

Comment:
Not sure if this is necessary after we move all `Frontend*Stmt`s to their own file.

Reply:
I don't think so, but currently there are some legacy stuff referring to the Frontend stmts, e.g. https://github.com/taichi-dev/taichi/blob/4d228b97361bedd5c7d88b0695a168d3ebe76d44/taichi/ir/ir.cpp#L933-L941

I think once these legacy constructs are removed, then it should be fine to remove this dependency..